src/constants.ts

```diff
@@ -86,3 +86,5 @@ var resources: any = {
 };
 
 _.set(window, 'OPENSHIFT_CONSTANTS.CATALOG_HELP_RESOURCES', resources);
+
+_.set(window, 'OPENSHIFT_CONSTANTS.NEW_PROJECT_MESSAGE', "Contact your system administrator at projectname@example.com to request a project.");
```

Comment:
It does use that, but defaults to what is in constants just in case. I can remove the constant if desired.

https://github.com/openshift/origin-web-catalog/blob/master/src/components/projects-summary/projects-summary.controller.ts#L56-L68